src/NServiceBus.Core/Routing/RoutingOptionExtensions.cs

```diff
@@ -32,13 +33,56 @@ public static void OverrideReplyToAddressOfIncomingMessage(this ReplyOptions opt
         }
 
         /// <summary>
-        /// Routes this message to the local endpoint instance.
+        /// Routes this message to any instance of this endpoint.
+        /// </summary>
+        /// <param name="option">Context being extended.</param>
+        public static void RouteToThisEndpoint(this SendOptions option)
+        {
+            option.Context.GetOrCreate<UnicastSendRouterConnector.State>()
+                .Option = UnicastSendRouterConnector.RouteOption.RouteToAnyInstanceOfThisEndpoint;
+        }
+
+        /// <summary>
+        /// Routes this message to this endpoint instance.
         /// </summary>
         /// <param name="option">Context being extended.</param>
-        public static void RouteToLocalEndpointInstance(this SendOptions option)
+        public static void RouteToThisInstance(this SendOptions option)
```

Comment:
Should we hide the scale out configuration options from the "99%" API then? The concept of endpoint vs. endpoint instance will confuse many users.